### ISSUE TITLE
Create new programming folders

### ISSUE THREAD

**Issue by amrs-tech** [amrs-tech/Hacktoberfest-Beginner-level#2]
Add an empty programming file to respective programming folder as `<programming-language>/empty.<file-extension>`

**Comment by jrajni** [comment]
can i take this?

**Comment by amrs-tech** [comment]
@jrajni Yeah sure, the issues are open to all. Sure, you too can contribute. Happy hacking :)
